packages/citation-js-utils/src/index.ts

```diff
@@ -189,15 +194,32 @@ export async function getCitations(bibtex: string): Promise<CitationRenderer> {
           },
           render(style?: CitationJSStyles) {
             return replaceUrlsWithAnchorElement(
-              cleanRef(cite.set(c).get({ ...defaultString, style: style ?? CitationJSStyles.apa })),
+              cleanRef(
+                cite.set(c).format('bibliography', {
```

Comment:
This (and the later usage) is an update to drop use of the deprecated `Cite.get` function.